virt/virt-2-5-release-notes.adoc

```diff
@@ -117,6 +117,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 * You can now import VMs with block-based storage into {VirtProductName}.
 
 //CNV-7296 - HCO and HPP CRs moved to v1beta1
+* The HyperConverged Operator (HCO), Containerized Data Importer (CDI), Hostpath Provisioner (HPP), and V2V custom resources have moved to API version `v1beta1`.
```

Comment:
Suggest changing "V2V" custom resources to "VM import" custom resources.

Reply:
Agreed, I would mention this component as *VM Import*, rather than "V2V".

@pkliczewski what do you think?

Reply:
Updated as suggested and included the apiVersion examples

Reply:
@fdupont-redhat a feedback from you will be welcomed.

Thanks

Reply:
Merging for a release note build for review. Feedback still welcome if there's anything wrong here. Thank you